pages/team.md

```diff
@@ -0,0 +1,83 @@
+# Team
+In our company, we are all equal. We only differ by responsibilities and salaries.
+The more we develop our professional skills, the more responsibilities we take, the higher the salary.
+
+We try to build a friendly atmosphere where everyone is important and every idea counts. Although we have general 
+team rules that we try to hold on to, we are always open to change them if the need arises. Better yet, we even break 
+those rules if a certain team member has a special case. Yep, that's right. We are all a team, but we also have 
+different personalities and needs.
+
+We can split our team in two major groups, part-time and full-time employees. 
+We will explain their differences down the road.
+
+## Teamwork
+That's probably our most beloved and powerful company asset.  
+We expect all members to help each other for problem solving, brainstorming ideas or sharing knowledge. The 
+time spent for teamwork is counted as effective work (explained below) and we strongly encourage it. This has 
+greatly benefited the professional development of our team and it boosts our daily morale.
+
+A more experienced developer helps a younger motivated teammate to learn all the required techniques that 
+we use on a daily basis. The lad quickly learns more and more, until he gets to help his previous teacher 
```

Comment:
The lad ?

Reply:
Pak confusing kjo sentence

Reply:
Kjo o si junior, ama perdoret ma shum neper libra ene artikuj. Mund ta bojm junior